lms/services/digest.py

```diff
@@ -56,12 +75,20 @@ def send_instructor_email_digests(
                 # We don't have an email address for this user.
                 continue
 
-            self._mailchimp_service.send_template(
-                "instructor-email-digest",
-                self._sender,
-                recipient=EmailRecipient(to_email, unified_user.display_name),
-                template_vars=digest,
-            )
+            try:
+                self._mailchimp_service.send_template(
+                    "instructor-email-digest",
+                    self._sender,
+                    recipient=EmailRecipient(to_email, unified_user.display_name),
+                    template_vars=digest,
+                )
+            except MailchimpError as err:
+                errors.append(err)
+                LOG.exception(err)
+                report_exception(err)
+
+        if errors:
```

Comment:
As mentioned in slack I reckon this PR could have the same value without this two lines.

We'd gather info about failures from mailchimp without potentially double-sending emails for the rest of the batch.

On the other hand this approach aims to be more resistant and go for "at least once"

Reply:
We actually won't double-send any emails as a result of this PR: retries aren't enabled yet for the `send_instructor_email_digests()` task on `main` or on this PR. https://github.com/hypothesis/lms/pull/5279/ enables retries and also deals with the double-sending issue.